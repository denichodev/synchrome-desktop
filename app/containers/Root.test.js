import React from 'react';
import Root from './Root';
import { configureStore, history } from '../store/configureStore';

const store = configureStore();

describe('Root', () => {
  it('should match its snapshot', () => {
    const component = shallow(
      <Root
        store={store}
        history={history}
      />
    );

    expect(component).toMatchSnapshot();
  });
});
