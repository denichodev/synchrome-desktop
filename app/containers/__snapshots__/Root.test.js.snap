// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root should match its snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <ConnectedRouter
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 2,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Component />
  </ConnectedRouter>
</Provider>
`;
